pkg/kube/client.go

```diff
@@ -1027,7 +1026,7 @@ func removeEmptyFiles(files []string) []string {
 
 func isEmptyFile(f string) bool {
 	fileInfo, err := os.Stat(f)
-	if err != nil {
+	if err != nil && !os.IsExist(err) {
```

Comment:
I think for other err, the fileInfo will be nil
And caling fileInfo.Size() will panic

Reply:
oh, yes, i thought if we need to judge the file exist, there are two conditons: one is err is nil, the other is err is isExist Error, but    the source code is only when err is not nil will return non-nil fileInfo